cleanlab/multiannotator.py

```diff
@@ -888,7 +937,13 @@ def get_majority_vote_label(
         annotator_agreement_with_consensus = nontied_labels_multiannotator.apply(
             lambda s: np.mean(s[pd.notna(s)] == nontied_majority_vote_label[pd.notna(s)]),
             axis=0,
-        ).to_numpy()
+        )
+
+        # impute average annotator accuracy for any annotator that do not overlap with consensus
+        mask = annotator_agreement_with_consensus.isna()
+        avg_annotator_agreement = np.mean(annotator_agreement_with_consensus[~mask])
+        annotator_agreement_with_consensus[mask] = avg_annotator_agreement
+
```

Comment:
do we have check somewhere if an entire column (i.e. annotator) contains all NAs?
Would probably be good to raise informative error message in that case

Reply:
Yup currently throws this error message: 
```
ValueError: labels_multiannotator cannot have columns with all NaN, each annotator must annotator at least one example.
Annotators {nan_annotators} did not label any examples.
```